### DISCUSSION TITLE
Subsample daa files

### DISCUSSION THREAD

**Discussion by arivers** [bbuchfink/diamond#484] [Ideas]
I think it would be helpful to have a flag to randomly subsample n records of an existing, un-Meganized .daa file using Diamond and return a new .daa file. The primary use case for this would be to optimize the LCA parameters in daa-meganize on a small dataset. Currently, I randomly sub-sample the fastq files then rerun the Diamond search but this is time-consuming for large databases.

**Comment by bbuchfink** [comment]
This goes towards being able to manipulate DAA files in a more general sense, which I plan to implement through an option to convert text-based alignment files to DAA. It may still take some time however until this is ready.
